TVShows/TVShows/Modules/ShowDetails/ShowDetailsViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  ShowDetailsViewController.swift
+//  TVShows
+//
+//  Created by Ivan Milicevic on 22/10/2018.
+//  Copyright (c) 2018 Ivan Milicevic. All rights reserved.
+//
+//  This file was generated by the 🐍 VIPER generator
+//
+
+import UIKit
+
+final class ShowDetailsViewController: UIViewController {
+
+    @IBOutlet private weak var showDetailsTableView: UITableView!{
+        didSet {
+            showDetailsTableView.dataSource = self
+            showDetailsTableView.delegate = self
+            showDetailsTableView.estimatedRowHeight = 100
+            showDetailsTableView.rowHeight = UITableView.automaticDimension
+            showDetailsTableView.separatorStyle = .none
+            showDetailsTableView.contentInset.bottom = 90
+        }
+    }
+
+    var presenter: ShowDetailsPresenterInterface!
+
+    // MARK: - Lifecycle -
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        presenter.viewDidLoad()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        navigationController?.setNavigationBarHidden(true, animated: animated)
```

Comment:
Missing super call! Always call super when you override something, expect when you are trying to avoid super implementation (very rare, especially on UI components)